project/Dependencies.scala

```diff
@@ -53,4 +54,6 @@ object Dependencies {
 
   val scalaTest = "org.scalatest" %% "scalatest" % versions.scalaTest % "test"
   val scalaCheck = "org.scalacheck" %% "scalacheck" % versions.scalaCheck % "test"
+
+  val rollupMetrics = "com.socrata" %% "rollup-metrics" % versions.rollupMetrics
```

Comment:
This is a private repo being depended on in a public repo. I'm not sure our policy here. We may consider opening that repo.

Reply:
Yeah good point, I'll definitely follow up on that.